disable any reference to (lib)gd, dvipng
thanks to Ondřej Surý <ondrej@debian.org>

this patch allows removing of embedded copies of libgd
completely. Before it was necessary to keep them only for the
configure step.

Thanks Ondřej!
---
 aclocal.m4          |    1 -
 libs/aclocal.m4     |    1 -
 m4/kpse-gd-flags.m4 |   31 -------------------------------
 m4/kpse-pkgs.m4     |    3 ---
 texk/aclocal.m4     |    1 -
 utils/aclocal.m4    |    1 -
 6 files changed, 38 deletions(-)

--- texlive-bin.orig/aclocal.m4
+++ texlive-bin/aclocal.m4
@@ -1187,7 +1187,6 @@
 m4_include([m4/kpse-common.m4])
 m4_include([m4/kpse-cxx-hack.m4])
 m4_include([m4/kpse-freetype2-flags.m4])
-m4_include([m4/kpse-gd-flags.m4])
 m4_include([m4/kpse-graphite2-flags.m4])
 m4_include([m4/kpse-harfbuzz-flags.m4])
 m4_include([m4/kpse-icu-flags.m4])
--- texlive-bin.orig/libs/aclocal.m4
+++ texlive-bin/libs/aclocal.m4
@@ -1187,7 +1187,6 @@
 m4_include([../m4/kpse-common.m4])
 m4_include([../m4/kpse-cxx-hack.m4])
 m4_include([../m4/kpse-freetype2-flags.m4])
-m4_include([../m4/kpse-gd-flags.m4])
 m4_include([../m4/kpse-graphite2-flags.m4])
 m4_include([../m4/kpse-harfbuzz-flags.m4])
 m4_include([../m4/kpse-icu-flags.m4])
--- texlive-bin.orig/m4/kpse-pkgs.m4
+++ texlive-bin/m4/kpse-pkgs.m4
@@ -33,7 +33,6 @@
 cairo
 pixman
 potrace
-gd
 freetype2
 libpng
 paper
@@ -57,7 +56,6 @@
 AC_REQUIRE([KPSE_PAPER_SYSTEM_FLAGS])[]dnl
 AC_REQUIRE([KPSE_LIBPNG_SYSTEM_FLAGS])[]dnl
 AC_REQUIRE([KPSE_FREETYPE2_SYSTEM_FLAGS])[]dnl
-AC_REQUIRE([KPSE_GD_SYSTEM_FLAGS])[]dnl
 AC_REQUIRE([KPSE_POTRACE_SYSTEM_FLAGS])[]dnl
 AC_REQUIRE([KPSE_PIXMAN_SYSTEM_FLAGS])[]dnl
 AC_REQUIRE([KPSE_CAIRO_SYSTEM_FLAGS])[]dnl
@@ -106,7 +104,6 @@
 dviljk
 dvipdf-x
 dvipdfm-x
-dvipng
 dvipos
 dvipsk
 dvisvgm
--- texlive-bin.orig/utils/aclocal.m4
+++ texlive-bin/utils/aclocal.m4
@@ -1187,7 +1187,6 @@
 m4_include([../m4/kpse-common.m4])
 m4_include([../m4/kpse-cxx-hack.m4])
 m4_include([../m4/kpse-freetype2-flags.m4])
-m4_include([../m4/kpse-gd-flags.m4])
 m4_include([../m4/kpse-graphite2-flags.m4])
 m4_include([../m4/kpse-harfbuzz-flags.m4])
 m4_include([../m4/kpse-icu-flags.m4])
--- texlive-bin.orig/m4/kpse-gd-flags.m4
+++ /dev/null
@@ -1,31 +0,0 @@
-# Public macros for the TeX Live (TL) tree.
-# Copyright (C) 2009-2013 Peter Breitenlohner <tex-live@tug.org>
-#
-# This file is free software; the copyright holder
-# gives unlimited permission to copy and/or distribute it,
-# with or without modifications, as long as this notice is preserved.
-
-# KPSE_GD_FLAGS
-# -------------
-# Provide the configure options '--with-system-gd' (if in the TL tree),
-# '--with-gd-includes', and '--with-gd-libdir'.
-#
-# Set the make variables GD_INCLUDES and GD_LIBS to the CPPFLAGS and
-# LIBS required for the `-lgd' library in libs/gd/ of the TL tree.
-AC_DEFUN([KPSE_GD_FLAGS], [dnl
-AC_REQUIRE([KPSE_CHECK_WIN32])[]dnl
-AC_REQUIRE([KPSE_LIBPNG_FLAGS])[]dnl
-AC_REQUIRE([KPSE_FREETYPE2_FLAGS])[]dnl
-_KPSE_LIB_FLAGS([gd], [gd], [],
-                [-IBLD/libs/gd/include], [BLD/libs/gd/libgd.a],
-                [test "x$kpse_cv_have_win32" = xno || GD_INCLUDES="$GD_INCLUDES -DBGDWIN32 -DNONDLL"],
-                [], [${top_builddir}/../../libs/gd/include/gd.h])[]dnl
-]) # KPSE_GD_FLAGS
-
-# KPSE_GD_OPTIONS([WITH-SYSTEM])
-# ------------------------------
-AC_DEFUN([KPSE_GD_OPTIONS], [_KPSE_LIB_OPTIONS([gd], [$1])])
-
-# KPSE_GD_SYSTEM_FLAGS
-# --------------------
-AC_DEFUN([KPSE_GD_SYSTEM_FLAGS], [_KPSE_LIB_FLAGS_SYSTEM([gd], [gd])])
--- texlive-bin.orig/texk/aclocal.m4
+++ texlive-bin/texk/aclocal.m4
@@ -1187,7 +1187,6 @@
 m4_include([../m4/kpse-common.m4])
 m4_include([../m4/kpse-cxx-hack.m4])
 m4_include([../m4/kpse-freetype2-flags.m4])
-m4_include([../m4/kpse-gd-flags.m4])
 m4_include([../m4/kpse-graphite2-flags.m4])
 m4_include([../m4/kpse-harfbuzz-flags.m4])
 m4_include([../m4/kpse-icu-flags.m4])
